librz/core/cmd/cmd_analysis.c

```diff
@@ -2698,38 +2593,27 @@ static void _analysis_calls(RzCore *core, ut64 addr, ut64 addr_end, bool printCo
 	free(block1);
 }
 
-RZ_API void rz_cmd_analysis_calls(RzCore *core, const char *input, bool printCommands, bool importsOnly) {
+RZ_API void rz_cmd_analysis_calls(RZ_NONNULL RzCore *core, bool print_commands, bool imports_only) {
```

Comment:
Maybe it's not `rz_cmd_*()` then? Probably `rz_core_analysis_calls()` is a better name. Also worth adding a brief description what it does.